chef/cookbooks/cinder/recipes/ceph.rb

```diff
@@ -17,51 +17,75 @@
 # Recipe:: ceph
 #
 
-glance_servers = search(:node, "roles:glance-server")
-if glance_servers.length > 0
-  glance_pool = glance_servers[0][:glance][:rbd][:store_pool]
-else   
-  glance_pool = nil
-end
+ceph_clients = {}
+ceph_keyrings = {}
 
 node[:cinder][:volumes].each_with_index do |volume, volid|
-  if volume['backend_driver'] == "rbd"
+  unless volume[:backend_driver] == "rbd"
+    next
+  else
+
+    ceph_conf = volume[:rbd][:config_file]
+    admin_keyring = volume[:rbd][:admin_keyring]
```

Comment:
For the record, like for the glance barclamp, I think that we should not use the attributes but hard-coded values when `volume['rbd']['use_crowbar']` is true.